src/main/java/core/basesyntax/reportservices/ReportCreatorImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.reportservices;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class ReportCreatorImpl implements ReportCreator {
+    @Override
+    public List<String> report(Map<String, Integer> transactions) {
+        List<String> report = new ArrayList<>();
+        report.add("fruit, quantity");
+        for (Map.Entry<String, Integer> entry : transactions.entrySet()) {
+            report.add(entry.getKey() + "," + entry.getValue());
```

Comment:
The entries in the report are being concatenated with a comma without any space. While this may be correct for CSV format, confirm that this is the desired format for the report. If a space is required after the comma, it should be added.